Notebooks/load_multiple_wells.ipynb

```diff
@@ -54,58 +54,226 @@
+      ],
       "text/plain": [
-       "<Figure size 1008x720 with 8 Axes>"
+       "<IPython.core.display.HTML object>"
       ]
      },
-     "metadata": {
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "import matplotlib.pyplot as plt\n",
+    "import welly.quality as qty\n",
     "\n",
-    "fig, axs = plt.subplots(figsize=(14, 10), ncols=len(p))\n",
+    "trial= { 'All': [qty.no_similarities],\n",
```

Comment:
Would it be useful to transform this into a function so that it can be reused? if so maybe create a new py file in dashdataviz